### ISSUE TITLE
Health Potions Regain Your Health Cap And Strength After You Die

### ISSUE THREAD

**Issue by Liham** [baxter-oop/AndOre#243]
Description: If you have a higher attack power and health cap than default because of potions and you die, your stats reset to default. However, if you use a healing potion after you've died, your health cap and attack power from before you died come back. 

Before and After Images of The Bug:
![biggest](https://cloud.githubusercontent.com/assets/22137914/20681588/445d96f8-b572-11e6-9484-68328dacf49d.PNG)
![biggerest](https://cloud.githubusercontent.com/assets/22137914/20681590/4801e3d6-b572-11e6-82ce-87172848ccb8.PNG)
![biggy](https://cloud.githubusercontent.com/assets/22137914/20681594/4a88e686-b572-11e6-8b65-9980be2baa54.PNG)
Image 1: Before Dying
Image 2: After Dying
Image 3: After Using Health Potion After Dying

Replications: First, make your health cap and attack power higher than normal. Next, buy a health potion. Then, kill yourself. Finally, use the health potion you bought, and your health cap and attack power will come back.

What World: Toivo

Email: liam.cs@baxter-academy.org

Date: 11/28/2016

**Comment by JCharante** [comment]
Yes I see, the Player.potions_taken were never reset after death, thank you for bringing this up.
